src/NzbDrone.Core/ImportLists/Trakt/List/TraktListRequestGenerator.cs

```diff
@@ -26,7 +26,7 @@ private IEnumerable<ImportListRequest> GetSeriesRequest()
         {
             var link = Settings.BaseUrl.Trim();
 
-            var listName = Settings.Listname.Trim();
+            var listName = Parser.Parser.ToUrlSlug(Settings.Listname.Trim());
```

Comment:
The trim here is unnecessary.

Reply:
are you sure, is listname getting trimmed elsewhere?

Reply:
The spaces get replaced with `-` then trimmed within `ToUrlSlug`, if we want to be more explicit we could `Trim()` within that function, but I think letting them be replaced and cleaned up is fine.